package.json

```diff
@@ -24,7 +24,7 @@
     "@graasp/apps-query-client": "github:graasp/graasp-apps-query-client#77-move-to-vite-for-bundling",
     "@graasp/pyodide": "github:graasp/graasp-pyodide",
     "@graasp/sdk": "0.9.3",
-    "@graasp/ui": "2.2.0",
+    "@graasp/ui": "^2.4.2",
```

Comment:
Prefer using pinned dependencies (instead of ranges such as the caret). So for instance, manually bump to e.g. 2.11.0 (or whatever is the latest)